components/table/ProductTable.tsx

```diff
@@ -32,10 +41,19 @@ const ProductTable = () => {
       </div>
       <div className='table-body'>
         <ProductTableHeader />
-        <div className='product-table-row-section'>
-          {productsList.map((product, key) => (
-            <ProductTableRow product={product} key={`product-${key}`} />
-          ))}
+        <div id='scrollableDiv' className='product-table-row-section'>
+          <InfiniteScroll
+            scrollableTarget='scrollableDiv'
+            dataLength={40}
+            next={handleLoadMoreData}
+            hasMore={hasMoreData}
+            loader={<p className='data-loader-message'>Loading more data...</p>}
+            style={{ overflowX: 'hidden' }}
+          >
+            {productsList.map((product, key) => (
```

Comment:
For SubTable we have SubTableRows and in here we're using iteration over productList to generate ProductTableRow, I think those two should be allined. Could you change ProductTableRow to ProductTableRows which will take productList as an input?

Reply:
Here is the difference. In ProductTableRow is actually one row, ProductTableRow component accept one product = one row, not list of products (SubTableRows accept list of building blocks).

Reply:
Thanks, as we discussed, these components will be updated later, during work on TECH-318.